pkg/policy/api/rule.go

```diff
@@ -94,9 +94,18 @@ var EntitySelectorMapping = map[Entity]EndpointSelector{
 //
 // - All members of this structure are optional. If omitted or empty, the
 //   member will have no effect on the rule.
-// - All members of this structure are evaluated independently, i.e. L4 ports
-//   allowed with ToPorts do not depend on a match of the FromEndpoints in the
-//   same IngressRule.
+//
+// - If multiple members are set, all of them need to match in order for
+//   the rule to take effect. The exception to this rule is FromRequires field;
+//   the effects of any Requires field in any rule will apply to all other
+//   rules as well.
+//
+// - For now, combining ToPorts, FromCIDR, and FromEndpoints in the same rule
```

Comment:
Is there a GitHub issue for this 'in the future' task? Please create one if it does not exist.

Reply:
Done: https://github.com/cilium/cilium/issues/1684